modules/carousel/jetpack-carousel.js

```diff
@@ -975,6 +981,27 @@ jQuery(document).ready(function($) {
 
 			return args.orig_file;
 		},
+		
+		getImageSizeParts: function(file, orig_width, isPhotonUrl) {
+			var size		= isPhotonUrl ? 
+							file.replace(/.*=([\d]+%2C[\d]+).*$/, '$1') : 
+							file.replace(/.*-([\d]+x[\d]+)\..+$/, '$1');
+						
+			var size_parts  = (size !== file) ? 
+							(isPhotonUrl ? size.split('%2C') : size.split('x')) :
+							[orig_width, 0];
+							
+			if (size_parts[0] === '9999')
+			{
+				size_parts[0] = '0';
+			}
+			if (size_parts[1] === '9999')
+			{
+				size_parts[1] = '0';
+			}
+			
```

Comment:
for the above if-statements, the opening curly brace does not need to be on a new line.  out of curiosity, where is 9999 coming from? it seems arbitrary.